<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="unitaires.Test_Communication" tests="1" skipped="0" failures="1" errors="0" timestamp="2021-07-08T16:55:31" hostname="ubuntu" time="0.516">
  <properties/>
  <testcase name="testBidirectionnal" classname="unitaires.Test_Communication" time="0.516">
    <failure message="java.lang.AssertionError" type="java.lang.AssertionError">java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:92)
	at org.junit.Assert.assertTrue(Assert.java:43)
	at org.junit.Assert.assertFalse(Assert.java:68)
	at org.junit.Assert.assertFalse(Assert.java:79)
	at unitaires.Test_Communication.testBidirectionnal(Test_Communication.java:85)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:45)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:42)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:28)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:30)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:263)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:68)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:47)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:231)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:60)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:50)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:222)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:300)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:119)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.base/java.lang.Thread.run(Thread.java:832)
</failure>
  </testcase>
  <system-out><![CDATA[[?[34m16:55:31.172 STDOUT (DEBUG) on Test worker org.gradle.api.internal.tasks.testing.processors.CaptureTestOutputTestResultProcessor.started(CaptureTestOutputTestResultProcessor.java:51)] ?[0m
[?[34m16:55:31.179 STDOUT (DEBUG) on Test worker utils.Log.init(Log.java:292)] ?[32mDémarrage du service de log?[0m
[?[34m16:55:31.183 STDOUT (DEBUG) on Test worker utils.Log.init(Log.java:293)] ?[0m?[0m
Initialisation du compteur...
[?[34m16:55:31.195 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:413)] Création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - everything.log' à '/home/ubuntu/TechTheTiller-HighLevel/logs/Thu Jul 08 16:55:31 UTC 2021 #44 - everything.log'?[0m
[?[34m16:55:31.197 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:417)] Réussite de la création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - everything.log?[0m
[?[34m16:55:31.206 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:421)] Ouverture du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - everything.log'?[0m
[?[34m16:55:31.213 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:413)] Création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44.log' à '/home/ubuntu/TechTheTiller-HighLevel/logs/Thu Jul 08 16:55:31 UTC 2021 #44.log'?[0m
[?[34m16:55:31.221 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:417)] Réussite de la création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44.log?[0m
[?[34m16:55:31.227 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:421)] Ouverture du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44.log'?[0m
[?[34m16:55:31.255 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:413)] Création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - errors.log' à '/home/ubuntu/TechTheTiller-HighLevel/logs/Thu Jul 08 16:55:31 UTC 2021 #44 - errors.log'?[0m
[?[34m16:55:31.260 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:417)] Réussite de la création du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - errors.log?[0m
[?[34m16:55:31.269 STDOUT (DEBUG) on Test worker utils.Log.attemptInitLogOutput(Log.java:421)] Ouverture du fichier de log './../logs/Thu Jul 08 16:55:31 UTC 2021 #44 - errors.log'?[0m
[?[34m16:55:31.302 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:57)] Creating socket waiting connection on port 10200?[0m
[?[34m16:55:31.339 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:60)] Receive buffer size on null is 65536?[0m
[?[34m16:55:31.353 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:71)] Waiting connection on port 10200?[0m
[?[34m16:55:31.357 COMMUNICATION (DEBUG) on SocketClient(localhost:10200) utils.communication.SocketClientInterface.lambda$init$0(SocketClientInterface.java:51)] Trying to connect to localhost on port 10200?[0m
[?[34m16:55:31.379 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:77)] Connection accepted on port 10200?[0m
[?[34m16:55:31.386 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:79)] Connection initialized on port 10200?[0m
[?[34m16:55:31.391 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:81)] Initialized connection with a 'ping'?[0m
[?[34m16:55:31.394 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:71)] Waiting connection on port 10200?[0m
[?[34m16:55:31.398 COMMUNICATION (DEBUG) on SocketClient(localhost:10200) utils.communication.SocketClientInterface.lambda$init$0(SocketClientInterface.java:53)] Socket created to localhost on port 10200?[0m
[?[34m16:55:31.401 COMMUNICATION (DEBUG) on SocketClient(localhost:10200) utils.communication.SocketClientInterface.lambda$init$0(SocketClientInterface.java:70)] SUCCESS to connect to localhost on port 10200?[0m
[?[34m16:55:31.544 COMMUNICATION (DEBUG) on Test worker utils.communication.SocketServerInterface.read(SocketServerInterface.java:107)] >> READ: R1?[0m
[?[34m16:55:31.669 STDOUT (DEBUG) on Finalizer utils.Log.close(Log.java:435)] ?[32mFermeture du service de log?[0m
[?[34m16:55:31.671 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]     	       _____?[0m
[?[34m16:55:31.672 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]               /     \?[0m
[?[34m16:55:31.676 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]               vvvvvvv  /|__/|?[0m
[?[34m16:55:31.677 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]                  I   /O,O   |?[0m
[?[34m16:55:31.679 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]                  I /_____   |      /|/|?[0m
[?[34m16:55:31.682 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]                 J|/^ ^ ^ \  |    /00  |    _//|?[0m
[?[34m16:55:31.687 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]                  |^ ^ ^ ^ |W|   |/^^\ |   /oo |?[0m
[?[34m16:55:31.688 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)]                   \m___m__|_|    \m_m_|   \mm_|?[0m
[?[34m16:55:31.696 COMMUNICATION (DEBUG) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:71)] Waiting connection on port 10200?[0m
[?[34m16:55:31.702 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)] ?[0m
[?[34m16:55:31.705 STDOUT (DEBUG) on Test worker org.gradle.api.internal.tasks.testing.processors.CaptureTestOutputTestResultProcessor.completed(CaptureTestOutputTestResultProcessor.java:80)]                 J|/^ ^ ^ \  |    /00  |    _//|?[0m
[?[34m16:55:31.712 STDOUT (DEBUG) on Finalizer utils.HLInstance.printMessage(HLInstance.java:357)] ------------------------- TOTORO'S END -------------------------?[0m
[?[34m16:55:31.721 STDOUT (DEBUG) on Test worker org.gradle.api.internal.tasks.testing.processors.CaptureTestOutputTestResultProcessor.completed(CaptureTestOutputTestResultProcessor.java:80)] ?[0m
]]></system-out>
  <system-err><![CDATA[[?[31m16:55:31.175 STDERR (CRITICAL) on Test worker] ?[0m
[?[31m16:55:31.670 STDERR (CRITICAL) on Thread-3] java.net.SocketException: Socket closed?[0m
[?[31m16:55:31.671 STDERR (CRITICAL) on Thread-3] 	at java.base/sun.nio.ch.NioSocketImpl.endAccept(NioSocketImpl.java:689)?[0m
[?[31m16:55:31.672 STDERR (CRITICAL) on Thread-3] 	at java.base/sun.nio.ch.NioSocketImpl.accept(NioSocketImpl.java:762)?[0m
[?[31m16:55:31.674 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:684)?[0m
[?[31m16:55:31.676 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.platformImplAccept(ServerSocket.java:650)?[0m
[?[31m16:55:31.679 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:626)?[0m
[?[31m16:55:31.682 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:583)?[0m
[?[31m16:55:31.683 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.accept(ServerSocket.java:540)?[0m
[?[31m16:55:31.684 STDERR (CRITICAL) on Thread-3] 	at utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:72)?[0m
[?[31m16:55:31.695 COMMUNICATION (CRITICAL) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:88)] Socket error: Socket closed?[0m
[?[31m16:55:31.702 STDERR (CRITICAL) on Thread-3] java.net.SocketException: Socket is closed?[0m
[?[31m16:55:31.703 STDERR (CRITICAL) on Thread-3] 	at java.base/java.net.ServerSocket.accept(ServerSocket.java:536)?[0m
[?[31m16:55:31.718 STDERR (CRITICAL) on Test worker] 	at java.base/java.net.ServerSocket.accept(ServerSocket.java:536)?[0m
[?[31m16:55:31.721 STDERR (CRITICAL) on Thread-3] 	at utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:72)?[0m
[?[31m16:55:31.723 COMMUNICATION (CRITICAL) on Thread-3 utils.communication.SocketServerInterface$1.run(SocketServerInterface.java:88)] Socket error: Socket is closed?[0m
[?[31m16:55:31.727 STDERR (CRITICAL) on Test worker] ?[0m
]]></system-err>
</testsuite>
